test/Conversion/TppToXsmm/tpp-to-xsmm-with-tiling.mlir

```diff
@@ -1,43 +0,0 @@
-// RUN: tpp-opt %s -convert-linalg-to-tpp="tile-sizes=32,32,32" -convert-tpp-to-xsmm -loop-invariant-code-motion -split-input-file | FileCheck %s -check-prefix=CONFIG1
```

Comment:
This looks like a bad test to begin with. It should have been actual tpp dialect to xsmm. Do we have a test like that?

Reply:
we do have them in test/Conversion/TppToXsmm